Engine/MenuAnnotation.cs

```diff
@@ -11,12 +11,25 @@ namespace OpenTap
     /// </summary>
     public sealed class MenuAnnotation : IOwnedAnnotation
     {
-        /// <summary> Creates a new instance. </summary>
-        /// <param name="member"> Which member of an object should the menu be generated for</param>
-        internal MenuAnnotation(IMemberData member) => this.member = member; 
+        /// <summary> Creates a 'member' menu annotation. </summary>
+        /// <param name="member"> Which member of an object should the menu be generated for.</param>
+        /// <param name="type"> Set this if menu items for the type should be generated. </param>
+        internal MenuAnnotation(IMemberData member, ITypeData type)
```

Comment:
This looks breaking!

Reply:
This is an internal API, so in the case we can change the argument types.